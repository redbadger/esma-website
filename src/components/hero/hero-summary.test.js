import React from "react";
import renderer from "react-test-renderer";

import HeroSummary from "./hero-summary";

describe("HeroSummary", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HeroSummary />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
