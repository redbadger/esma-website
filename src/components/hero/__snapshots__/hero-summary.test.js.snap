// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroSummary renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "h1eao5",
      "next": undefined,
      "styles": "
  display:flex;justify-content:center;width:100%;@media (min-width: 1024px){flex-direction:column;height:100%;width:42rem;}

  > div {
    width:22rem;margin-top:2rem;@media (min-width: 768px){width:44rem;margin-top:4rem;}@media (min-width: 1024px){width:28rem;margin-top:0;margin-left:5rem;}@media (min-width: 1280px){width:36rem;margin-left:10rem;}
  }

  .summaryText {
    --text-opacity:1;color:rgba(45, 49, 66, var(--text-opacity));font-size:1.25rem;line-height:1.75rem;margin-bottom:2rem;@media (min-width: 768px){font-size:2rem;line-height:2.75rem;margin-bottom:3rem;}
  }

  span.highlight {
    --bg-opacity:1;background-color:rgba(255, 186, 8, var(--bg-opacity));font-weight:600;
  }
",
      "toString": [Function],
    }
  }
>
  <div>
    <div
      className="summaryText"
    >
      <span
        className="highlight"
      >
        Social inequality
      </span>
       is persistent and systematic. Often where you start in life is where youâ€™ll end up.
      <br />
      <br />
       Together we can change this.
    </div>
    <button
      css={
        Object {
          "map": undefined,
          "name": "vxeuw9",
          "next": undefined,
          "styles": "
  height:3rem;width:22rem;--bg-opacity:1;background-color:rgba(45, 49, 66, var(--bg-opacity));--text-opacity:1;color:rgba(255, 255, 255, var(--text-opacity));font-weight:700;
",
          "toString": [Function],
        }
      }
      onClick={[Function]}
    >
      Find out how
    </button>
  </div>
</div>
`;
